backend/src/Designer/Services/Implementation/RepositorySI.cs

```diff
@@ -846,8 +847,8 @@ public List<ServiceResource> GetServiceResources(string org, string repository,
 
             foreach (FileSystemObject resourceFile in resourceFiles)
             {
-                Stream fs = File.OpenRead($"{repopath}/{resourceFile.Path}");
-                ServiceResource serviceResource = System.Text.Json.JsonSerializer.Deserialize<ServiceResource>(fs, new System.Text.Json.JsonSerializerOptions() { PropertyNamingPolicy = System.Text.Json.JsonNamingPolicy.CamelCase });
+                string jsonString = File.ReadAllText($"{repopath}/{resourceFile.Path}");
+                ServiceResource serviceResource = JsonConvert.DeserializeObject<ServiceResource>(jsonString);
```

Comment:
Is there a reason you change form system.text? Is this newton?

Reply:
Yes this is newton. I noticed that is was already used in RepositorySI so I started using it in the PUT-method. And after that I also changed it in the GET-method so make them more alike.